docs/apm-data-security.asciidoc

```diff
@@ -92,6 +93,21 @@ which means the list of sanitized fields can be updated without needing to redep
 * Python: {apm-py-ref-v}/configuration.html#config-capture-body[`capture_body`]
 * Ruby: {apm-ruby-ref-v}/configuration.html#config-capture-body[`capture_body`]
 
+[discrete]
+[[filters-personal-data]]
+==== Personal data
+
+By default, the APM Server captures some personal data associated with trace events:
+
+* `client.ip`: The client's IP address. Typically derived from the HTTP headers of incoming requests.
+`client.ip` is also used in conjunction with the {ref}/geoip-processor.html[`geoip` processor] to assign
+geographical information to trace events. To learn more about how `client.ip` is derived,
```

Comment:
https://github.com/elastic/apm-server/issues/6423 mentions

> how client IP is connected to determining client IPs (i.e. through geoIP enrichment)

Am I understanding this correctly, @axw?

Reply:
Yup!

Reply:
:facepalm: should have said "determining client geo"